content/open-source/apko/faq.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "FAQs"
```

Comment:
```suggestion
title: "apko FAQs"
```

I put our pages through an SEO tool and it gets cranky when we have more than one page with the same name